lib/timeline/timeline.dart

```diff
@@ -67,6 +67,8 @@ class TimelineScreen extends Screen {
       ..disabled = true
       ..click(_resumeRecording);
 
+    // TODO(kenzie): is this right extension for this button? Doesn't match the
+    // IntelliJ 'Show widget rebuild information' checkbox.
     final trackWidgetBuildsButton =
```

Comment:
It is the right extension. We need some word smithing here and intellij as the intellij checkbox name is a bit long for how we are showing the UI in devtools currently.

Reply:
Ah, okay. Will remove todo.